modules/core/src/main/java/org/apache/ignite/mxbean/DataRegionMetricsMXBean.java

```diff
@@ -124,6 +124,26 @@
     @MXBeanDescription("Offheap used size in bytes.")
     @Override public long getOffheapUsedSize();
 
+    /** {@inheritDoc} */
+    @MXBeanDescription("Gets memory page size.")
+    @Override public int getPageSize();
+
+    /** {@inheritDoc} */
+    @MXBeanDescription("Gets total size of pages loaded to the RAM. When persistence is disabled, this metric is equal TotalAllocatedSize.")
+    @Override public long getPhysicalMemorySize();
+
+    /** {@inheritDoc} */
+    @MXBeanDescription("Gets used checkpoint buffer size in pages.")
+    @Override public long getUsedCheckpointBufferPages();
+
+    /** {@inheritDoc} */
+    @MXBeanDescription("Gets used checkpoint buffer size in bytes.")
```

Comment:
_The_ used checkpoint buffer size in pages.

Reply:
The used checkpoint buffer size in **bytes**.